_commands/tools/scp.md

```diff
@@ -0,0 +1,33 @@
+---
+---
+
+scp
+-------
+
+`scp` is used to make a _secure copy_ of files or directories to and from a remote host. Similar to "cp" except either the source or destination should be a remote host. You'll want to run this command from your local machine.
```

Comment:
Why will I want to run this command from my local machine? Works just as well the other way around as your example denotes.